src/content/advanced/forcing-cgroupv1/index.md

```diff
@@ -0,0 +1,97 @@
+---
+linkTitle: force nodes cgroup v1
+title: Forcing nodes to use legacy Control Groups v1
+description: This article describes how to force nodes to use legacy Control Groups v1 instead the default v2.
+weight: 60
+menu:
+  main:
+    parent: advanced
+user_questions:
+ -  How can I enable cgroupsv1?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+---
+
+# Forcing cgroups v1
+
+{{< platform_support_table aws="beta=v17.0.0" azure="beta=v17.0.0" >}}
+
+## Introduction
+
+Flatcar version `3033.2.0` and above uses Control Groups v2 by default, which means all nodes will be using cgroups v2 for all kubernetes containers, as opposed to Control Groups v1 used on previous versions.
+
+To ensure a smooth transition, in case our customers need time to modify applications to make them compatible with Control Groups v2, we introduced a mechanism that will allow using Control Groups v1 on specific node pools.
+
+In order to apply the changes, roll of the nodes is required.
+
+We recommend set the proper the annotation on the node pool before updating to the release `17.0.0`.
```

Comment:
I'd add these two lines after the `kubectl annotate` line, or they are easy to miss. WDYT?